jupyter_rfb/_colab.py

```diff
@@ -0,0 +1,31 @@
+try:
+    # determine if running in colab
+    from google.colab import output
+    from google.colab.output._widgets import _installed_url
+except Exception:
```

Comment:
Probably want to be more specific:

```suggestion
except ImportError:
```